Mathlib/Analysis/InnerProductSpace/l2Space.lean

```diff
@@ -380,26 +382,31 @@ namespace HilbertBasis
 instance {ι : Type*} : Inhabited (HilbertBasis ι 𝕜 ℓ²(ι, 𝕜)) :=
   ⟨ofRepr (LinearIsometryEquiv.refl 𝕜 _)⟩
 
+open Classical in
 /-- `b i` is the `i`th basis vector. -/
 instance instCoeFun : CoeFun (HilbertBasis ι 𝕜 E) fun _ => ι → E where
   coe b i := b.repr.symm (lp.single 2 i (1 : 𝕜))
 
+open Classical in
```

Comment:
use decidability assumptions?

Reply:
I added them - this makes the proofs require `convert rfl`, though.